Longer constants for word sizes

On big mailboxes with lots of messages selected,
16k characters can be less than required.  Users 
were unable to delete large numbers of messages.

This patch just increases the size of MAXWORD
and MAXQUOTED.

Index: cyrus-imapd-2.3.12rc1/imap/imapparse.c
===================================================================
--- cyrus-imapd-2.3.12rc1.orig/imap/imapparse.c	2008-03-24 13:09:17.000000000 -0400
+++ cyrus-imapd-2.3.12rc1/imap/imapparse.c	2008-04-09 07:49:00.000000000 -0400
@@ -53,8 +53,8 @@
 #include "exitcodes.h"
 
 enum {
-    MAXQUOTED = 32768,
-    MAXWORD = 32768,
+    MAXQUOTED = 524288,
+    MAXWORD = 524288,
     MAXLITERAL = INT_MAX / 20
 };
 
